lib/Config.js

```diff
@@ -958,6 +985,40 @@ class Config extends EventEmitter {
             config.reportToken ||
             uuid.v4().toString();
         this.reportEndpoint = process.env.REPORT_ENDPOINT;
+
+        // External backends
+        // Currently supports configuring httpAgent(s) for keepAlive
+        this.externalBackends = defaultExternalBackendsConfig;
+        if (config.externalBackends) {
+            const extBackendsConfig = Object.keys(config.externalBackends);
+            extBackendsConfig.forEach(b => {
+                // assert that it's a valid backend
+                assert(validExternalBackends[b] !== undefined,
+                    `bad config: ${b} is not one of valid external backends: ` +
+                    `${Object.keys(validExternalBackends).join(', ')}`);
+
+                const { httpAgent } = config.externalBackends[b];
+                assert(typeof httpAgent === 'object',
+                    `bad confg: ${b} must have httpAgent object defined`);
```

Comment:
typo: `confg` => `config`
same typos  in `asserts` below

Reply:
Fixed.